### DISCUSSION TITLE
Scene creation using Blender

### DISCUSSION THREAD

**Discussion by weirdooo-18** [NVlabs/sionna#386] [Q&A]
Hey there!!
I was trying to create a scene using Blender and OSM. After exporting the scene to Sionna in .xml format, I am getting an error relating to Plane.ply. I have followed all the steps as per the scene creation tutorial, and the plane(ground) is also included in the scene. Can you please help me out with this? 

 Thank you.

![sss](https://github.com/NVlabs/sionna/assets/161753847/8adb17f5-5ed9-42cb-a7cb-e911dc441ea8)

**Comment by jhoydis** [comment]
Hi @weirdooo-18,

Could it be that you have only copied the .xml file from the export and not the folder called `meshes` which contains the .ply files?

**Reply by weirdooo-18** [reply]
The same error persists even if the meshes folder is uploaded along with the .xml file.

**Reply by jhoydis** [reply]
Hm, did you check that the meshes folder contains the file `Plane.ply`? 
Which version of Blender and the Plugin are you using?

**Comment by merlinND** [comment]
Hello @weirdooo-18,

Someone recently had the same symptom, but it seems that re-doing the export and making sure to keep the `meshes` directory together with the XML file was enough to resolve the problem:
https://github.com/mitsuba-renderer/mitsuba-blender/issues/93

**Comment by xslobo05** [comment]
Hello @weirdoo-18,

First of all, do not use Blender 4.0, use version 3.6.9. Version 4.0 did not work for me both on Win and Macos.

All objects must be type "Mesh" (triangle icon). The error you're getting is probably caused by this. To change the type, right click on the object -> Convert to -> Mesh.

Second, make sure when exporting to check box that says "Export IDs". Also don't forget to set axis correctly (Y forward, Z up).

Before loading the scene with load_scene() make sure the "meshes" folder is in the same directory as the XML file.

I hope that helps.

**Reply by merlinND** [reply]
Thanks a lot for sharing this advice @xslobo05, I hope these steps will work for @weirdooo-18.

**Reply by xslobo05** [reply]
No problem. I'm doing raytracing on custom scenes imported from Blender and ran into this at the beginning. I believe the .PLY file type is specific to Mesh. Also, the folder is called "meshes" so it makes sense.
